test/ILLink.RoslynAnalyzer.Tests/DynamicallyAccessedMembersAnalyzerTests.cs

```diff
@@ -537,6 +538,8 @@ public class TestSystemTypeBase : Type
 
 		public override string Name => throw new NotImplementedException ();
 
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors
+			| DynamicallyAccessedMemberTypes.NonPublicConstructors)]
```

Comment:
Indentation is off - you added one with spaces, while the rest of the code here uses tabs (even in the string literals).